src/redux/actions/concepts/dictionaryConcepts.js

```diff
@@ -80,18 +108,18 @@ export const fetchDictionaryConcepts = (
   page = 1,
 ) => async (dispatch, getState) => {
   dispatch(isFetching(true));
-  let url = `${conceptType}/${conceptOwner}/collections/${conceptName}/concepts/?q=${query}&limit=${limit}&page=${page}&verbose=true`;
+  let url = `${conceptType}/${conceptOwner}/sources/${conceptName}/concepts/?q=${query}&limit=${limit}&page=${page}&verbose=true`;
```

Comment:
Why did we move from collections to sources?

Reply:
Unless otherwise, I think this was a bug. I have shared this concern on the OCL OpenMRS Talk channel.

Reply:
@kmunene @alexmochu  I noticed that sources  returned the latest version of the concepts in a dictionary

Reply:
This was discussed starting fromhttps://talk.openmrs.org/t/developing-the-ocl-for-openmrs-application/17771/218?u=alexmochu and ending at https://talk.openmrs.org/t/developing-the-ocl-for-openmrs-application/17771/221?u=alexmochu . The resolution was that the above url should remain as it is. So this change was not supposed to happen.

Reply:
I've made the update.